package.json

```diff
@@ -62,21 +61,20 @@
   },
   "devDependencies": {
     "@babel/cli": "^7.8.3",
-    "@types/node": "^13.9.0",
-    "@types/ramda": "github:types/npm-ramda#dist",
-    "@types/react": "^16.9.23",
-    "@types/react-dom": "^16.9.5",
+    "@types/node": "^13.11.0",
+    "@types/react": "^16.9.32",
+    "@types/react-dom": "^16.9.6",
     "@types/react-helmet": "^5.0.15",
     "@types/react-router-dom": "^5.1.3",
     "babel-plugin-i18next-extract": "^0.6.1",
     "clsx": "^1.1.0",
```

Comment:
Can be removed

Reply:
`"pretty-quick": "^2.0.1",`

```
"husky": {
    "hooks": {
      "pre-commit": "pretty-quick --staged && lint-staged"
    }
  },
  "lint-staged": {
    "src/**/*.{js,jsx,ts,tsx}": [
      "npm run lint"
    ]
  },
```

Reply:
`"source-map-explorer": "^2.4.1"`

`"analyze": "source-map-explorer 'build/static/js/*.js'",`